src/components/Main.tsx

```diff
@@ -0,0 +1,202 @@
+                fill="var(--icon-gray-color)"
+                stroke="var(--icon-gray-color)"
+                strokeWidth="2"
+                className="three-dots-svg"
+              >
+                <circle cx="12" cy="5" r="1"></circle>
+                <circle cx="12" cy="12" r="1"></circle>
+                <circle cx="12" cy="19" r="1"></circle>
+              </svg>
+            </div>
+            <div className="recently-used-card card-base">
+              <span className="square-with-shadow square"></span>
+              <div className="card-circle-container self-center">
+                <span className="circle-span"></span>
+                <span className="circle-span circle-span-offset"></span>
+              </div>
+              <div className="card-text flex-col justify-center">
+                <h3>Project: fitbit</h3>
+                <p>Created: 28.02.2020</p>
+              </div>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="18"
+                height="18"
+                viewBox="0 0 24 24"
+                fill="var(--icon-gray-color)"
+                stroke="var(--icon-gray-color)"
+                strokeWidth="2"
+                className="three-dots-svg"
+              >
+                <circle cx="12" cy="5" r="1"></circle>
+                <circle cx="12" cy="12" r="1"></circle>
+                <circle cx="12" cy="19" r="1"></circle>
+              </svg>
+            </div>
+            <div className="recently-used-card card-base">
+              <span className="square-with-shadow square"></span>
+              <div className="card-circle-container self-center">
+                <span className="circle-span"></span>
+                <span className="circle-span circle-span-offset"></span>
+                <span className="circle-span circle-span-offset flex-center justify-center">
+                  +3
+                </span>
+              </div>
+              <div className="card-text flex-col justify-center">
+                <h3>Client Documents</h3>
+                <p>Created: 4.03.2020</p>
+              </div>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="18"
+                height="18"
+                viewBox="0 0 24 24"
+                fill="var(--icon-gray-color)"
+                stroke="var(--icon-gray-color)"
+                strokeWidth="2"
+                className="three-dots-svg"
+              >
+                <circle cx="12" cy="5" r="1"></circle>
+                <circle cx="12" cy="12" r="1"></circle>
+                <circle cx="12" cy="19" r="1"></circle>
+              </svg>
+            </div>
+          </div>
+        </section>
+        <section className="recent-files">
+          <div className="recent-files-header flex justify-between">
+            <h2 className="recent-files-title title">Recent Files</h2>
+            <p>View All</p>
```

Comment:
This must be a button right?

Reply:
Yes definitely a button, I will keep an eye on semantic tags for future projects